src/config/browser-options.js

```diff
@@ -181,7 +182,23 @@ function buildBrowserOptions(defaultFactory, extra) {
 
         prepareBrowser: options.optionalFunction("prepareBrowser"),
 
-        screenshotsDir: options.stringOrFunction("screenshotsDir"),
+        screenshotsDir: option({
+            defaultValue: defaultFactory("screenshotsDir"),
+            validate: value => {
+                if (!_.isString(value) && !_.isFunction(value)) {
+                    throw new Error('"screenshotsDir" must be a string or function');
+                }
+            },
+            map: (value, _, __, { isSetByUser }) => {
+                const deprecatedScreensPath = "hermione/screens";
+
+                if (!isSetByUser && fs.existsSync(deprecatedScreensPath) && !fs.existsSync(value)) {
+                    return deprecatedScreensPath;
+                }
+
+                return value;
+            },
```

Comment:
If user uses default value, we check for "hermione/screens" folder. If it exists, and "testplane/screens" does not, switch to "hermione/screens"

Only works when default value is untouched, and new default directory does not exist, but old one - does